vm/src/actor_state.rs

```diff
@@ -48,6 +50,17 @@ impl<'de> Deserialize<'de> for ActorState {
     }
 }
 
+impl Default for ActorState {
+    fn default() -> Self {
+        Self {
+            code: Cid::default(),
+            state: Cid::default(),
+            balance: BigUint::zero(),
+            sequence: 0,
+        }
+    }
+}
```

Comment:
I don't personally like having the actor state have default because it would allow the "undefined" cids to potentially be used without noticing. If previous suggestion is done, there isn't a need for this